src/manifest.json

```diff
@@ -50,6 +50,7 @@
         "downloads",
         "find",
         "history",
+        "search",
```

Comment:
We ought to add something to AMO.md explaining the new permission.

Reply:
I'm not seeing anything about search engines on https://support.mozilla.org/en-US/kb/permission-request-messages-firefox-extensions , do you know what the AMO definition will be?